### DISCUSSION TITLE
shouldnt be needed but stating it anyways

### DISCUSSION THREAD

**Discussion by geekmoto13** [magicbug/Cloudlog#892] [General]
Who knows who might find this useful, but It would be nice if the time label on the live qso page said Time: UTC or had some indicator that it was UTC. now i just today got cloudlog up and running on a raspberry pi inside my home network as i dont plan on using it outside my home, but does anyone know where that label is generated from (what file) as I would like to update it to read "Time: UTC" just so i can look at that field and be able to gleen what time its showing, as even though i have hamclock, i am not used to doing the time calculation in a reasonable amount of time in my head without looking it up

**Comment by magicbug** [comment]
Cloudlog defaults to UTC which is the standard time used within the hobby so I just took it as read everyone would be able to deal with that without needing to show it as it takes up space that isn't always available.

If you want to edit your own then check out the code inside /application/views

**Reply by geekmoto13** [reply]
Thanks yeah I can’t seem to do the math in my head fast enough to do the calculations for my local time and then freak out thinking i have blacked out for a period of time

**Reply by geekmoto13** [reply]
> 
> 
> Cloudlog defaults to UTC which is the standard time used within the hobby so I just took it as read everyone would be able to deal with that without needing to show it as it takes up space that isn't always available.
> 
> If you want to edit your own then check out the code inside /application/views

um would it be possible to get the label "time" in the qso to read UTC time so others who have a hard time with calculating the offset, in their heads. so the line 
`<label for="start_time">UTC <?php echo $this->lang->line('general_word_time'); ?></label>`
just gets the UTC added near the beginning of the line,

That's the change i was suggesting simple, doesn't affect pagination or layout and simple update

**Comment by geekmoto13** [comment]
Nevermind, found it
